redaxo/src/core/lib/response.php

```diff
@@ -539,6 +539,24 @@ public static function sendCookie($name, $value, array $options = [])
         header($str, false);
     }
 
+    /**
+     * @param string $name    The name of the cookie
```

Comment:
```suggestion
     * Clear the given cookie by name. You might pass additional options in case the name is not unique or the cookie is not stored on the current domain.
     *
     * @param string $name    The name of the cookie
```